R/test.R

```diff
@@ -64,16 +99,10 @@ runTests <- function(appDir=".", filter=NULL){
       warning("You've disabled `shiny.autoload.r` via an option but this is not passed through to shinytest. Consider using a _disable_autoload.R file as described at https://rstd.io/shiny-autoload")
     }
 
-    sares <- shinytest::testApp(appDir)
-    res <- list()
-    lapply(sares$results, function(r){
-      e <- NA_character_
-      if (!r$pass){
-        e <- simpleError("Unknown shinytest error")
-      }
-      res[[r$name]] <<- e
-    })
-    return(structure(list(result=all(is.na(res)), files=res), class="shinytestrun"))
+    return(do.call(rbind, lapply(shinytest::testApp(appDir)[["results"]], function(r) {
```

Comment:
Instead of calling `shinytest::testApp`, can we make this just use the tests/shinytest.R script? The idea is that we wouldn't hard-code each type of testing, but let the user provide scripts to run each type of test.
For example: https://github.com/rstudio/shinycoreci-apps/blob/master/apps/041-dynamic-ui/tests/shinytest.R

This may

Reply:
We surely can, are you suggesting that we don't attempt to detect or special-case shinytest at all? If so, 👍

Reply:
Discussed in person with @wch and we realized this branch was for dealing with old shinytest behavior that populated tests/ with shinytest-specific code.